### DISCUSSION TITLE
大家觉得新图标好看吗？

### DISCUSSION THREAD

**Discussion by 88558639** [gkd-kit/gkd#673] [Polls]
我个人觉得有点抽象

**Comment by BennettChina** [comment]
原本那版就挺好的，简约美。

**Reply by zetaloop** [reply]
换成旧版图标的版本：[zetaloop/gkd-replace-icon](https://github.com/zetaloop/gkd-replace-icon) 😄
<img height="50px" src="https://raw.githubusercontent.com/zetaloop/gkd-replace-icon/main/theme.png">

**Reply by lisonge** [reply]
> 换成旧版图标的版本：[zetaloop/gkd-replace-icon](https://github.com/zetaloop/gkd-replace-icon) 😄

@zetaloop https://github.com/orgs/gkd-kit/discussions/679

**Reply by zetaloop** [reply]
> > 换成旧版图标的版本：[zetaloop/gkd-replace-icon](https://github.com/zetaloop/gkd-replace-icon) 😄
> 
> @zetaloop [github.com/orgs/gkd-kit/discussions/679](https://github.com/orgs/gkd-kit/discussions/679)

了解了，我会撤回发布版，仅保留修改器。

**Reply by BennettChina** [reply]
> > > 换成旧版图标的版本：[zetaloop/gkd-replace-icon](https://github.com/zetaloop/gkd-replace-icon) 😄
> > 
> > 
> > @zetaloop [github.com/orgs/gkd-kit/discussions/679](https://github.com/orgs/gkd-kit/discussions/679)
> 
> 了解了，我会撤回发布版，仅保留修改器。

谢谢，我Mac平台的用不了，fork你的改了下。

**Comment by lisonge** [comment]
源代码是完全开放的，如果觉得不好看，建议 fork 后更改代码里的图标，不一定非要用官方这版

此外修改版请遵守 https://github.com/orgs/gkd-kit/discussions/679

**Comment by rsysm** [comment]
看起来像一位故人🤔

**Comment by TerryZ** [comment]
奥哟，你干嘛，这样的图标被人发现了多不好

**Comment by weierz** [comment]
我觉得很好看，头下面又像背带裤又像篮球，很抽象，设计者很有水平。
<img width="254" alt="image" src="https://github.com/user-attachments/assets/1025e937-23cb-447f-8ca2-e0982b3314fc">

**Comment by ccav14** [comment]
用了新图标之后应该改名CXK
